Dfe.Academies.External.Web.UnitTest/Controllers/SchoolControllerTests.cs

```diff
@@ -3,59 +3,84 @@
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Logging;
 using Moq;
+using Moq.Protected;
 using NUnit.Framework;
+using System;
+using System.IO;
 using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
 using System.Threading.Tasks;
 
 namespace Dfe.Academies.External.Web.UnitTest.Controllers;
 
 [Parallelizable(ParallelScope.All)]
 internal sealed class SchoolControllerTests
 {
-    /// <summary>
-    /// TODO MR:- re-test after PR55 - plumb in trams API
-    /// </summary>
-    //[Test]
-    public async Task SchoolController___Search___ReturnsResult()
+    [Test]
+    public async Task Seach___ResultsFound___ResultsReturned()
```

Comment:
ReferenceDataRetrievalService.SearchSchools(schoolSearch) returns null in this test

Reply:
plumbed in mock IHttpClientFactory which has fixed this